migration-engine/connectors/sql-migration-connector/src/lib.rs

```diff
@@ -284,12 +295,26 @@ impl MigrationConnector for SqlMigrationConnector {
     fn validate_migrations<'a>(
         &'a mut self,
         migrations: &'a [MigrationDirectory],
+        namespaces: Option<Namespaces>,
     ) -> BoxFuture<'a, ConnectorResult<()>> {
         Box::pin(async move {
-            self.flavour.sql_schema_from_migration_history(migrations, None).await?;
+            self.flavour
+                .sql_schema_from_migration_history(migrations, None, namespaces)
+                .await?;
             Ok(())
         })
     }
+
+    fn extract_namespaces(&self, schema: &DatabaseSchema) -> Option<Namespaces> {
+        let sql_schema: &SqlDatabaseSchema = schema.downcast_ref();
+        Namespaces::from_vec(
+            &mut sql_schema
+                .describer_schema
+                .namespace_walkers()
```

Comment:
could be caching, or it could be because clippy is rebasing somehow. There were both `namespace_walkers()` and `walk_namespaces()` and they did the same things so I deleted one of them this week. Can't remember which one.